include/User.hpp

```diff
@@ -3,6 +3,13 @@
 
 # include <string>
 
+typedef enum	e_user_flag
+{
+	 CHANNEL_USER,
+	 CHANNEL_OPERATOR,
+	 CHANNEL_CREATOR
+}				t_user_flag;
+
```

Comment:
This should be *user_status* instead of *user_flag*. Flags would be invisible (`i`), ...